.golangci.yml

```diff
@@ -119,7 +119,8 @@ linters:
     - usestdlibvars
     - varnamelen
     - whitespace
-
```

Comment:
qq: reason for disabling this?

Reply:
It's disabled cause it expects a defer Span.End() in the same function as StartFunctionSpan() and throws an error regarding memory leak. This sometimes causes improper span handling, which is why I explicitly end the span in the outer function calls.